pkg/binder/cache/cache.go

```diff
@@ -217,3 +227,7 @@ func (cache *binderCache) FindStore(storeName commonstore.StoreName) commonstore
 	defer cache.mu.RUnlock()
 	return cache.CommonStoresSwitch.Find(storeName)
 }
+
+func (cache *binderCache) List() []framework.NodeInfo {
+	return append(cache.nodeSlices.InPartitionNodeSlice.Nodes(), cache.nodeSlices.OutOfPartitionNodeSlice.Nodes()...)
```

Comment:
Need lock.

BTW it's dangerous to expose all of the nodes.

Reply:
Yes, I have add lock.

Because the binder needs all the nodeInfo information when performing topology checks for `InterPodAffinity` and `PodTopologySpread`, I added this function. Is there a safer method?